distro/common/knot.service

```diff
@@ -10,6 +10,49 @@ User=knot
 Group=knot
 CapabilityBoundingSet=CAP_NET_BIND_SERVICE CAP_SETPCAP
 AmbientCapabilities=CAP_NET_BIND_SERVICE CAP_SETPCAP
+# Require systemd >= v238
+#TemporaryFileSystem=/run:ro /var:ro
+# Require systemd >= v233
+#BindPaths=/run/systemd
+#BindPaths=/run/knot
```

Comment:
I think using `RuntimeDirectory=` would be better. Then, `knot.tmpfiles` file would no longer be needed for systems that have this option.

Reply:
With that we could also drop `knot.tmpfiles`.

Reply:
Note: `*Directory` options need >= v235.

Reply:
That would be a better solution.